lesson07/assignment/spaceland/src/components/AttractionsPage.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+<div class="attractions">
+  <h2 class="text-danger text-center m-4 p-5">{{ msg }}</h2>
+
+  <body>
+    <main class="main">
+      <div class="d-flex flex-wrap justify-content-center">
+        <img class="img-fluid img-thumbnail m-2 w-25" v-for="image in images" v-bind:key="image" :src="image.url" :alt="image.alt" />
+      </div>
+
+      <div class="d-flex flex-wrap justify-content-center p-3 m-5 border border-warning">
+        <h2 class="d-flex display-5 p-3 text-center align-items-center">The Orbital</h2>
+        <p class="display-5 text-center">Are you ready to be spun at the velocity of the Earth? This ride is great for people who love thrills and speed. Each spaceship has a max capacity of two passengers. You must be at least 5' 6" to ride. Please
+          make sure to avoid eating at least two hours prior to riding.</p>
+      </div>
```

Comment:
This would also be good content to include as variables, that you could display with a v-for loop